packages/components/src/circular-option-picker/index.tsx

```diff
@@ -58,6 +73,34 @@ export function Option( {
 	);
 }
 
+export function OptionGroup( {
+	className,
+	options,
+	...additionalProps
+}: OptionGroupProps ) {
+	const compositeState = useContext( CircularOptionPickerContext );
+
+	// This is unlikely to happen, but on the off-chance that we've ended up
+	// with a list of `OptionGroup`s, we will just return those instead.
+	if ( Array.isArray( options ) && options[ 0 ] instanceof OptionGroup ) {
+		return <>options</>;
```

Comment:
Let's make sure this works visually, and also add a unit test. I don't think we can use `instanceof` in this way.

```suggestion
	if ( Array.isArray( options ) && options[ 0 ] instanceof OptionGroup ) {
		return <>{ options }</>;
```